packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -4,6 +4,13 @@ import colors from 'kleur';
 import { posixify } from '../../utils/filesystem.js';
 import { write_if_changed } from './utils.js';
 
+/** @type {import('typescript')} */
+// @ts-ignore
+let ts = undefined;
+try {
+	ts = (await import('typescript')).default;
```

Comment:
I think we might need to import relative to the cwd? We ran into this before with `adapter-auto` — it's possible, though rare, for `typescript` to be installed in a package while `@sveltejs/kit` is installed in a workspace root

Reply:
We're doing the same in `write_types` and I heard no complaints so far. Still do it?

Reply:
ah, yeah that's probably fine in that case. though we may as well DRY it out